hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/Gateway.java

```diff
@@ -54,6 +56,7 @@ public Void call() throws Exception {
     TracingUtil.initTracing("S3gateway", ozoneConfiguration);
     OzoneConfigurationHolder.setConfiguration(ozoneConfiguration);
     UserGroupInformation.setConfiguration(ozoneConfiguration);
+    checkSecurityAndSCMHAEnabled(ozoneConfiguration);
```

Comment:
S3G does not talk to SCM directly? We can skip the check here.

Reply:
I have added the check, because when secure SCM HA is enabled, to not to start services in ozone.

Reply:
The other check makes sense because they are related to SCM. But S3G does not need to know SCM HA + security is enabled. The S3G node itself may not need to upgrade with the SCM HA change.

Reply:
When other services failed to start, even starting S3G is of no real use when OM and SCM are down in the cluster.

>The S3G node itself may not need to upgrade with the SCM HA change.
I am fine with removing the check here.